po/fa.po

```diff
@@ -17511,58 +17511,58 @@ msgstr ""
 
 #: src/glossary.md:32
 msgid "argument:"
-msgstr ""
+msgstr "Argument: Information passed to a function or method when it is called."
```

Comment:
Wow, thanks for filling out all of these!

Please note that the process is a bit different: you need to modify this Markdown file to fill out the missing English definitions: https://github.com/google/comprehensive-rust/blob/main/src/glossary.md.

Make sure to follow the special style I used there:

```markdown
- term:\
  Explanation of the term.
```

That will ensure that the `term:` part is highlighted in bold and that the explanation comes on the next line. It will also ensure that each term is separated out into its own individually translatable message here!

After that is merged, the English `msgid` fields you see here will update naturally as part of the next [translation refresh](https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md#refreshing-an-existing-translation).

Reply:
sure